docs/source/using.rst

```diff
@@ -167,30 +169,32 @@ over and dependencies are installed into the environment.  The
 environments are stored in the ``env`` directory so that the next time
 the benchmarks are run, things will start much faster.
 
-.. note::
-
-    The easiest way to benchmark in multiple versions of Python is to
-    install `anaconda <https://store.continuum.io/cshop/anaconda/>`__
-    or `miniconda <http://conda.pydata.org/miniconda.html>`__
-    installation, and make sure the ``conda`` command is available on
-    your ``PATH``.
-
-    If not using Conda, virtualenv will be used.  In this case,
-    ``asv`` does not build Python interpreters for you, but it expects
-    to find each of the Python versions specified in the
-    ``asv.conf.json`` file available on the ``PATH``.  For example, if
-    the ``asv.conf.json`` file has::
-
-        "pythons": ["2.7", "3.3"]
-
-    then it will use the executables named ``python2.7`` and
-    ``python3.3`` on the path.  There are many ways to get multiple
-    versions of Python installed -- your package manager, ``apt-get``,
-    ``yum``, ``MacPorts`` or ``homebrew`` probably has them, or you
-    can also use `pyenv <https://github.com/yyuu/pyenv>`__.  ``asv``
-    always runs its benchmarks in a virtual environment, so it will
-    not change what is installed in any of the python environments on
-    your system.
+Environments can be created using different tools.  By default,
+``asv`` ships with support for `anaconda
+<https://store.continuum.io/cshop/anaconda/>`__ and `virtualenv
+<https://pypi.python.org/pypi/virtualenv>`__, though plugins may be
+installed to support other environment tools.  By default, if
+``conda`` is found in your ``PATH``, it will be used, and if not, fall
+back to ``virtualenv``.  To override this behavior, the
```

Comment:
Hmm... puzzling.  That's exactly what `asv` does to check the existence of `conda`.  It then goes on to do:

```
$ conda create --yes -p /tmp/foo python=2.7 --dry-run
```

Does that fail?

If that's not it, all I can suggest is to try to step through `plugins/conda.py:Conda:matches` to see why it's returning `False`.